runtime/op/join/ztests/kinds.yaml

```diff
@@ -1,12 +1,12 @@
 script: |
   echo === ANTI ===
-  zq -z 'anti join (file B.zson) on a=b | sort a' A.zson
+  zq -z 'anti join (file B.zson) on a=b' A.zson
   echo === LEFT ===
-  zq -z 'left join (file B.zson) on a=b hit:=sb | sort a' A.zson
+  zq -z 'left join (file B.zson) on a=b hit:=sb' A.zson
   echo === INNER ===
-  zq -z 'inner join (file B.zson) on a=b hit:=sb | sort a' A.zson
+  zq -z 'inner join (file B.zson) on a=b hit:=sb' A.zson
   echo === RIGHT ===
-  zq -z 'right join (file C.zson) on b=c hit:=sb | sort c' B.zson
```

Comment:
While reviewing, I happened to notice that the original actually looks broken here, since the `| sort` are being tacked on _after_ the `join`. It looks like it only worked consistently because the inputs happened to be pre-sorted by the respective keys.

Reply:
Well spotted! I'll back out the change to this file since these sorts aren't before the join.